controllers/servers/csi/csi_controller_server.py

```diff
@@ -77,6 +77,7 @@ def CreateVolume(self, request, context):
 
                 if volume_parameters.virt_snap_func and source_id:
                     array_mediator.validate_space_efficiency_matches_source(space_efficiency, source_id, source_type)
+                    array_mediator.validate_required_bytes_matches_source(required_bytes, source_id, source_type)
```

Comment:
we can remove these two from the interface, and instead let the server utils compare the volumes using `get_object_by_id`

Reply:
done